src/api/createDevice/src/v1/steps.py

```diff
@@ -22,7 +23,8 @@ def parse_device_payload(data, cache) -> CreateDeviceIncomingParams:
 def create_device(data, cache) -> Device:
     product: CpmProduct = data[read_product]
     payload: CreateDeviceIncomingParams = data[parse_device_payload]
-    return product.create_device(**payload.dict())
+    environment = data[read_environment]
+    return product.create_device(env=environment, **payload.dict())
```

Comment:
I thought it would work as `environment.create_product` and then `product.create_device` (and environment would be implicitly passed in)

Reply:
But why environment.create_product? The environment is not on the product level. Also ticket just states it's an ENUM, not another layer of classes.